packages/vitest/src/node/workspace.ts

```diff
@@ -315,12 +315,26 @@ export class WorkspaceProject {
 
   getSerializableConfig() {
     const optimizer = this.config.deps?.optimizer
+    const poolOptions = this.config.poolOptions
+
     return deepMerge({
       ...this.config,
       coverage: this.ctx.config.coverage,
 
-      pool: this.ctx.config.pool,
-      poolOptions: this.ctx.config.poolOptions,
+      poolOptions: {
+        forks: {
+          ...this.ctx.config.poolOptions?.forks,
```

Comment:
Should we actually spread the root config here? As mentioned on https://github.com/vitest-dev/vitest/issues/4731#issuecomment-1852921150, projects do not inherit `pool` and `poolOptions` from root configuration. 

And as project level configuration does not allow setting `pool.<name>.isolate`, does this mean that when using workspaces, there is no way to configure `isolate` at all?